actions/changelog-generate/README.md

```diff
@@ -0,0 +1,88 @@
+
+Additional configuration options can be explored [here](https://github.com/mikepenz/release-changelog-builder-action#configuration-specification).
+
+## Input Parameters
+
+| Name                    | Required |           Default Value           | Description                                                                                                                                                  |
+| ----------------------- | :------: | :-------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| changelog-file          |    ✅    |                 -                 | Path to the Changelog.md file                                                                                                                                |
+| github-token            |    ✅    |                 -                 | The GitHub token for committing the changes                                                                                                                  |
+| new-tag                 |    ✅    |                 -                 | New version                                                                                                                                                  |
+| commitMode              |    ❌    |              "false"              | Special configuration for projects which work without PRs. Uses commit messages as changelog. This mode looses access to information only available for PRs. |
+| config                  |    ❌    | "./.github/changelog-config.json" | Path to the changelog config JSON file                                                                                                                       |
+| fetchReleaseInformation |    ❌    |              "false"              | Will enable fetching additional release information from tags.                                                                                               |
+| fetchReviewers          |    ❌    |              "false"              | Will enable fetching the users/reviewers who approved the PR.                                                                                                |
+| old-tag                 |    ❌    |                 -                 | Previous version                                                                                                                                             |
+| output                  |    ❌    |           "changes.md"            | Relative path to a file to store the resulting changelog in.                                                                                                 |
```

Comment:
I don't see why this is still needed. If you require a temporary file, I'd suggest creating one in `/tmp`